addons/flashdrive/CfgVehicles.hpp

```diff
@@ -0,0 +1,29 @@
+class CfgVehicles 
+{
+	/* ================================================================================ */
+	// Flash Drive
+	class Land_USB_Dongle_01_F;
+	class FlashDrive_AE3: Land_USB_Dongle_01_F
```

Comment:
We should keep the original class name and the add "_AE3", like in all other cases.